core-web/libs/sdk/client/src/lib/client/sdk-js-client.ts

```diff
@@ -319,16 +321,24 @@ export class DotCmsClient {
 
             // Format the URL correctly depending on the 'path' value
             const formattedPath = path === '/' ? '/' : `/${path}`;
-            const url = `${this.config.dotcmsUrl}/api/v1/nav${formattedPath}${
+            const url = `${this.#config.dotcmsUrl}/api/v1/nav${formattedPath}${
                 queryParams ? `?${queryParams}` : ''
             }`;
 
-            const response = await fetch(url, this.requestOptions);
+            const response = await fetch(url, this.#requestOptions);
 
             return response.json();
         }
     };
 
+    static init(config: ClientConfig): DotCmsClient {
+        return this.instance ?? (this.instance = new DotCmsClient(config));
```

Comment:
Let's throw a console.warn if they try to init twice.

Reply:
It's already Implemented and tested!